iquip/apps/builder.py

```diff
@@ -587,3 +604,33 @@ def onSubmitted(self, rid: int):
     def frames(self) -> Tuple[Tuple[str, BuilderFrame]]:
         """Overridden."""
         return (("", self.builderFrame),)
+
+class _ScanEntry(QWidget):
+    """Entry class for a Scannable argument.
+    
+    Attributes:
+        name: The name of the experiment.
```

Comment:
Is it the experiment name? I think it is the name of the scannable object.

Reply:
It is the name of experiment.
It was added because later version created new widget for scannable object display, and experiment name should be added again.

Reply:
Then, please fix it.

Reply:
I missed it while changing lacoation of the Scan class in the codes

Reply:
Did you fix it?